src/backend/src/routes/projects.routes.ts

```diff
@@ -63,6 +63,11 @@ projectRouter.post(
   nonEmptyString(body('pdmFileName')).optional(),
   ProjectsController.createAssembly
 );
+projectRouter.post(
+  '/bom/material/:materialId/assign-assembly',
+  nonEmptyString(body('assemblyId')),
+  ProjectsController.assignMaterialAssembly
```

Comment:
I think we wanna include `validInputs` in the endpoint if it accepts at least one prop from its payload. What is our thought here?

Reply:
Yes, you're right! Totally forgot about it, good catch